src/gateways/Database/index.js

```diff
@@ -14,7 +14,7 @@ export default (() => {
           options.connectionString = process.env.TEST_DATABASE_URL;
         }
 
-        if (process.env.NODE_ENV !== "test") {
+        if (process.env.NODE_ENV === "production") {
```

Comment:
Just checking this is the correct way around?

If it's production we almost certainly want to check the SSL cert? I may have missed something

Reply:
Ye this change means that if the env is production, require SSL.

If we're only requiring SSL in production then I guess we can remove the `{ rejectUnauthorized: false }` and just replace it right `true` ? (Was this added to make it work with dev signed certs ?)